staff/eduardo-wiater/precourse/bingo/index.js

```diff
@@ -0,0 +1,262 @@
+"use strict" 
+function bingo(){
+   
+  var jugadorActivo = {}  
+  var bombo = [];
+  var contadorMatch = 0 , contador = 0 , puntos = 100 , estadoLinea = false;
```

Comment:
I suggest the whole name in english like counterMatch or countMatch.

Reply:
OK! done.
Thank you ! Happy code

Reply:
Ops.. i did it wrong.

Reply:
done